string robotname
---
float64 x
float64 y
float64 th
